cypress/integration/boards.spec.js

```diff
@@ -4,47 +4,113 @@ import dataBoard from "../fixtures/data.json"
 
 describe("create organization", () => {
 
-    beforeEach("Login user", () => {
+    beforeEach("User needs to be login", () => {
         cy.visit("/login")
         cy.get(loginModel.email).type(dataBoard.user.email)
         cy.get(loginModel.password).type(dataBoard.user.pass)
         cy.get(loginModel.logInBtn).click();
 
+        cy.get('[class=vs-u-text--uppercase]')
+            .should('be.visible')
+            .and('contain', 'My Organizations')
+
+        cy.get('span.el-dropdown-link').should(($loggedInUser) => {
+            expect($loggedInUser).to.contain('Maja Cveticanin')
+        })
+        cy.get('.vs-c-modal__body > .el-button > .el-icon-close').click()
```

Comment:
What does this do? Close the modal? If so, could you please move it to POM and make a selector out of it?